Sources/SwiftMetricsShim/MetricHandlers.swift

```diff
@@ -87,7 +87,7 @@ class SwiftHistogramMetric: RecorderHandler, SwiftMetric {
 
 class SwiftSummaryMetric: TimerHandler, SwiftMetric {
     
-    var metricName: String
+    private(set) var metricName: String
```

Comment:
Do those ever need to be set though? 🤔 

I don't see any other modifications of them other than during initialization, am I missing something?

Reply:
I checked setting them to let - can't be done due to protocol conformance 😞

Reply:
Hm? which conformance?

I only see:

```
protocol SwiftMetric {
    var metricName: String { get }
    var metricType: MetricType { get }
    init(name: String, labels: [String: String], meter: Meter)
}
```

Which are get-only properties so they can be witnessed by `let metricName` and `let metricType`.

I even checked locally and yea, seems just fine, do you have something else in mind?

Reply:
I've written Swift for half a decade and I've never realised that a `let variableName` is valid for a protocol which defines `var variableName` - today I learned!

What I was referring to is that I tried changing the `var` within the protocol definition itself to a `let` and that doesn't work. What I didn't realise did work is changing the `var` to a `let` in each of the classes but leaving the protocol unaffected. I had naively assumed that it would flag up an inconsistency between class and conformance.

Apologies for being so dense, great shout and has now been changed.

Thanks for educating me!

Reply:
Ha, okey :-) Lovely, happy to help.

Yes it's a neat pattern to implement get only requirements with a `let`, very happy to have spread the knowlage about this a bit more then 👍